packages/client/src/modules/blocks/data-blocks/form-editor/openFormEditablePage.tsx

```
@@ -0,0 +1,105 @@
+function unpatchSchemaToolbars(schema) {
+  const patch = (node: ISchema) => {
+    if (!node || typeof node !== 'object') return;
+    if (node['x-uid']) {
+      node['x-uid'] = uid();
+    }
+    if (node['x-toolbar'] === 'FormItemSchemaToolbar') {
+      node['x-toolbar'] = 'EditableFormItemSchemaToolbar';
+    }
+    if (node['x-component'] === 'CardItem' && node['x-toolbar'] === 'BlockSchemaToolbar') {
+      node['x-toolbar'] = 'EditableFormToolbar';
+      delete node['x-settings'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)